Example/SlideController/VerticalController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  VerticalController.swift
+//  SlideController_Example
+//
+//  Created by Pavel Kondrashkov on 10/17/17.
+//  Copyright © 2017 Touchlane LLC. All rights reserved.
+//
+
+import UIKit
+import SlideController
+
+class VerticalController {
+    private let internalView = VerticalView()
+    private let slideController: SlideController<VerticalTitleScrollView, VerticalTitleItem>
+    
+    var optionsController: (ViewAccessible & ContentActionable)? {
+        didSet {
+            internalView.optionsView = optionsController?.view
+            
+            optionsController?.removeDidTapAction = removePage
+            optionsController?.insertDidTapAction = insertPage
+            optionsController?.appendDidTapAction = appendPage
```

Comment:
removePage, insertPage and appendPage are functions that use strong self reference. It can cause retain cycle